code/modules/modular_computers/computers/subtypes/preset_wristbound.dm

```diff
@@ -33,49 +33,54 @@
 	_app_preset_type = /datum/modular_computer_app_presets/civilian
 
 /obj/item/modular_computer/handheld/wristbound/preset/advanced/cargo
-	icon_state = "wristbound_cargo"
 	_app_preset_type = /datum/modular_computer_app_presets/supply
+	icon_state = "wristbound-cargo"
```

Comment:
This icon state doesn't look to be in the DMI. Is this supposed to be `wristbound-sup`?

Reply:
Nice catch, thanks.